ray-on-gke/tpu/kuberay-tpu-webhook/main.go

```diff
@@ -413,36 +506,45 @@ func extractPod(admissionReview *admissionv1.AdmissionReview) (*corev1.Pod, erro
 }
 
 // add DNS hostname and TPU_WORKER_ID env var to the Pod
-func mutatePod(admissionReview *admissionv1.AdmissionReview) (*admissionv1.AdmissionResponse, error) {
+func mutatePod(podLister listersv1.PodLister, admissionReview *admissionv1.AdmissionReview) (*admissionv1.AdmissionResponse, error) {
```

Comment:
`mutatePod` can be a method of `TPUWebhookServer`. Remove the podLister parameter and call it from the receiver

Reply:
[dce1bbe](https://github.com/GoogleCloudPlatform/ai-on-gke/pull/723/commits/dce1bbe4b09fd0a48304cb2d519414578a3ecdc0)